packages/mc-scripts/config/create-webpack-config-for-production.js

```diff
@@ -100,91 +101,84 @@ module.exports = ({
   entryPoint,
   sourceFolders,
   toggleFlags = defaultToggleFlags,
-}) => ({
-  // Don't attempt to continue if there are any errors.
-  bail: true,
+}) => {
+  // NOTE: Without properly defauling passing one flag (e.g. `enabledVendorOptimizations`)
+  // would disable other flags defaulted to `true` (e.g. `enableExtractCss`).
+  const defaultedToggleFlags = {
+    ...defaultToggleFlags,
+    ...toggleFlags,
+  };
```

Comment:
I think we can simplify this like:

```js
const mergedToggleFlags = Object.assign({}, defaultToggleFlags, toggleFlags);
```

...and remove the default named parameter "toggleFlags ~~= defaultToggleFlags~~,"

...and we don't need the comment anymore.

Reply:
Prettier will format it to spreading syntax generally. Will see what happens if the arg is not defaulted to an object.